<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>&lt;h4&gt;Enter Your Text here : &lt;/h4&gt;
</string>
      </property>
      <property name="buddy">
       <cstring>textEdit_input</cstring>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_input"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>&lt;h4&gt;Enter The Key (1-25): &lt;/h4&gt;
</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_key</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_key">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_encrypt">
        <property name="styleSheet">
         <string notr="true">font: 75 11pt &quot;Ubuntu&quot;;
background-color: rgb(143, 240, 164);
font-size: 24px;</string>
        </property>
        <property name="text">
         <string>Encrypt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_output"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
